devicetypes/smartthings/aeon-multisensor.src/aeon-multisensor.groovy

```diff
@@ -85,12 +86,14 @@ metadata {
 		valueTile("battery", "device.battery", inactiveLabel: false, decoration: "flat", width: 2, height: 2) {
 			state "battery", label:'${currentValue}% battery', unit:""
 		}
-		standardTile("configure", "device.configure", inactiveLabel: false, decoration: "flat", width: 2, height: 2) {
-			state "configure", label:'', action:"configuration.configure", icon:"st.secondary.configure"
-		}
 
 		main(["motion", "temperature", "humidity", "illuminance"])
-		details(["motion", "temperature", "humidity", "illuminance", "battery", "configure"])
+		details(["motion", "temperature", "humidity", "illuminance", "battery"])
+	}
+
+	preferences {
+		input name: "timeout", type: "number", title: "Motion timeout", description: "Enter guard interval before receiving another motion event, in seconds", range: "20..60", defaultValue: 20
+		input name: "period", type: "number", title: "Reporting period", description: "Enter reporting period, in minutes", range: "4..15", defaultValue: 5
```

Comment:
You should mention the range in the description text, like "Enter guard interval before receiving another motion event, in seconds (20-60)"